src/django_peerctl/signals.py

```diff
@@ -54,3 +59,41 @@ def set_session_device(sender, **kwargs):
         return
 
     session.device = session.port.object.device_id
+
+
+@receiver(pre_save, sender=EmailTemplate)
+def change_default_email_template(sender, **kwargs):
+    """
+    Before a EmailTemplate is saved we want to update the
+    default field to False for all other EmailTemplates of
+    the same type
+    """
+
+    email_template = kwargs.get("instance")
+
+    if not email_template:
+        return
+
+    if email_template.default:
+        EmailTemplate.objects.filter(type=email_template.type).exclude(
+            pk=email_template.pk
```

Comment:
email_template.pk wont be set if its a newly created email template - not sure if this would break, but definitely should test if this works correctly when setting default on an email template during creation.

Reply:
tested and it works as intended.